src/java.base/share/classes/java/util/zip/ZipOutputStream.java

```diff
@@ -378,6 +378,10 @@ public synchronized void write(byte[] b, int off, int len)
      * Finishes writing the contents of the ZIP output stream without closing
      * the underlying stream. Use this method when applying multiple filters
      * in succession to the same output stream.
+     * <p>
```

Comment:
People reading this not too carefully may think the combined length referrs to the concatenated string length. I know "bytes" is the last word, so technically this is correct. But maybe being slightly more explicit that the length depends on the chosen charset would remove some ambiguity?